src/mono/mono.proj

```diff
@@ -81,4 +86,64 @@
 
   <Target Name="Test" />
 
+  <!-- Copy Mono runtime bits to the local .dotnet dir for local experiments (temp solution) -->
+  <Target Name="PatchLocalDotnet" DependsOnTargets="ValidateTestHost">
+    <PropertyGroup>
+      <LocalDotnetRuntimeDir>$([System.IO.Directory]::GetDirectories("$(LocalDotnetDir)\shared\Microsoft.NETCore.App")[0])</LocalDotnetRuntimeDir>
+    </PropertyGroup>
+    <ItemGroup>
+      <_MonoRuntimeArtifacts Include="$(BinDir)\*.*" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(_MonoRuntimeArtifacts)"
+        DestinationFolder="$(LocalDotnetRuntimeDir)"
+        SkipUnchangedFiles="true" />
+  </Target>
+
+  <Target Name="PatchTestHost" DependsOnTargets="ValidateTestHost">
+    <ItemGroup>
+      <_MonoRuntimeArtifacts Include="$(BinDir)\*.*" />
+    </ItemGroup>
+    <Copy SourceFiles="@(_MonoRuntimeArtifacts)"
+          DestinationFolder="$(TestHostPath)\shared\Microsoft.NETCore.App\$(ProductVersion)" 
+          SkipUnchangedFiles="true"/>
+  </Target>
+
+  <!-- Run netcore\sample\HelloWorld sample using Mono Runtime -->
```

Comment:
I am not sure it's worth it. I'd prefer to have better documentation for this instead

Reply:
@marek-safar um.. what exactly is not worth it? -- having the RunSample target?

Reply:
yeah, I'd much rather see documentation with working steps for `dotnet new ....` to runt on Mono

Reply:
Wouldn't this have temporary value until we figure that part out?  I know some people like to use RunSample as an easy way to test their stuff.

Reply:
I guess we figure it out already if we can run the sample which is form of `dotnet new console`

Reply:
I see what you mean.  Yeah, I agree.

Reply:
it seems most of us use HelloWorld for experiments/fixes/improvements. Do you suggest to remove it and let everybody do:
```
dotnet new console
dotnet publish -c Release -r osx-x64
copy libcoreclr.dylib & SPC.dll by hands to the publish directory
```
instead of `make run-sample` ?

Currently `run-sample` doesn't require patching local .dotnet (which imo should be avoided if possible)

Reply:
yep, I think having this document something is more valuable than having it in Makefile (although you can keep sample of this example if you think it's valuable)

Reply:
I've simplified all the things so now runsample uses local dotnet. But I personally would prefer to keep it at least for a while.